zh/tidb-toolkit.md

```diff
@@ -174,94 +242,119 @@ tidb-ctl schema in mysql
     helm init --upgrade
     ```
 
-Kubernetes 应用在 Helm 中被打包为 chart。PingCAP 针对 Kubernetes 上的 TiDB 部署运维提供了多个 Helm chart：
+3. 配置 Helm repo
 
-* `tidb-operator`：用于部署 TiDB Operator；
-* `tidb-cluster`：用于部署 TiDB 集群；
-* `tidb-backup`：用于 TiDB 集群备份恢复；
-* `tidb-lightning`：用于 TiDB 集群导入数据；
-* `tidb-drainer`：用于部署 TiDB Drainer；
-* `tikv-importer`：用于部署 TiKV Importer；
+    Kubernetes 应用在 Helm 中被打包为 chart。PingCAP 针对 Kubernetes 上的 TiDB 部署运维提供了多个 Helm chart：
 
-这些 chart 都托管在 PingCAP 维护的 helm chart 仓库 `https://charts.pingcap.org/` 中，你可以通过下面的命令添加该仓库：
+    * `tidb-operator`：用于部署 TiDB Operator；
+    * `tidb-cluster`：用于部署 TiDB 集群；
+    * `tidb-backup`：用于 TiDB 集群备份恢复；
+    * `tidb-lightning`：用于 TiDB 集群导入数据；
+    * `tidb-drainer`：用于部署 TiDB Drainer；
+    * `tikv-importer`：用于部署 TiKV Importer；
 
-{{< copyable "shell-regular" >}}
+    这些 chart 都托管在 PingCAP 维护的 helm chart 仓库 `https://charts.pingcap.org/` 中，你可以通过下面的命令添加该仓库：
 
-```shell
-helm repo add pingcap https://charts.pingcap.org/
-```
+    {{< copyable "shell-regular" >}}
+
+    ```shell
+    helm repo add pingcap https://charts.pingcap.org/
+    ```
 
-添加完成后，可以使用 `helm search` 搜索 PingCAP 提供的 chart：
+    添加完成后，可以使用 `helm search` 搜索 PingCAP 提供的 chart：
 
-如果 Helm 版本 < 2.16.0:
+    如果 Helm 版本 < 2.16.0:
 
-{{< copyable "shell-regular" >}}
+    {{< copyable "shell-regular" >}}
 
-```shell
-helm search pingcap -l
-```
+    ```shell
+    helm search pingcap -l
+    ```
 
-如果 Helm 版本 >= 2.16.0:
+    如果 Helm 版本 >= 2.16.0:
 
-{{< copyable "shell-regular" >}}
+    {{< copyable "shell-regular" >}}
 
-```shell
-helm search pingcap -l --devel
-```
+    ```shell
+    helm search pingcap -l --devel
+    ```
 
-```
-NAME                    CHART VERSION   APP VERSION DESCRIPTION
-pingcap/tidb-backup     v1.0.0                      A Helm chart for TiDB Backup or Restore
-pingcap/tidb-cluster    v1.0.0                      A Helm chart for TiDB Cluster
-pingcap/tidb-operator   v1.0.0                      tidb-operator Helm chart for Kubernetes
-...
-```
+    ```
+    NAME                    CHART VERSION   APP VERSION DESCRIPTION
+    pingcap/tidb-backup     v1.0.0                      A Helm chart for TiDB Backup or Restore
+    pingcap/tidb-cluster    v1.0.0                      A Helm chart for TiDB Cluster
+    pingcap/tidb-operator   v1.0.0                      tidb-operator Helm chart for Kubernetes
+    ...
+    ```
```

Comment:
```suggestion
        ```
        NAME                    CHART VERSION   APP VERSION DESCRIPTION
        pingcap/tidb-backup     v1.0.0                      A Helm chart for TiDB Backup or Restore
        pingcap/tidb-cluster    v1.0.0                      A Helm chart for TiDB Cluster
        pingcap/tidb-operator   v1.0.0                      tidb-operator Helm chart for Kubernetes
        ...
        ```
```